en-US/Docs/PinMappingsRPi.md

```diff
@@ -203,8 +203,10 @@ public async void I2C()
     // 0x40 is the I2C device address
     var settings = new I2cConnectionSettings(0x40);
 
-    // Create an I2cDevice with the default controller and specified I2C settings
-    using (I2cDevice device = await I2cDevice.GetDefault(settings))
```

Comment:
Yes, this looks like an error.  The method we are encouraging can be found [here](https://github.com/ms-iot/samples/blob/23d8e2d8fbc9d1819c02527cc24582f769082e26/WeatherStation/CS/WeatherStation/StartupTask.cs):

```
            var controller = await I2cController.GetDefaultAsync();
            sensor = controller.GetDevice(new I2cConnectionSettings(0x40));
```